### PR TITLE
Update the help menu

### PR DESCRIPTION
**Pull request by fabienrohrer**
Implement proposal discussed in #873

Implemented the following menu:

Capture d’écran 2019-09-10 à 14 04 49

Doc is compliant with this modification:
- https://cyberbotics.com/doc/guide/the-user-interface#help-menu
- https://cyberbotics.com/doc/guide/general-faq

### CODE REVIEW

**Comment by fabienrohrer** [comment]
👍

File: src/webots/core/WbStandardPaths.cpp [hunk 1]
```diff
@@ -138,6 +138,11 @@ const QString &WbStandardPaths::cyberboticsUrl() {
   return url;
 };
 
+const QString &WbStandardPaths::githubRepositoryUrl() {
+  static QString url("https://github.com/cyberbotics/webots");
```
Comment (omichel) [inline src/webots/core/WbStandardPaths.cpp]:
```suggestion
  static const QString url("https://github.com/cyberbotics/webots");
```

File: src/webots/gui/WbMainWindow.cpp [hunk 2]
```diff
@@ -908,23 +908,63 @@ QMenu *WbMainWindow::createHelpMenu() {
...
-  action->setStatusTip(tr("Report a bug to Cyberbotics."));
+  action->setStatusTip(tr("Report a bug to the GitHub repository."));
   connect(action, &QAction::triggered, this, &WbMainWindow::openBugReport);
   menu->addAction(action);
 
   action = new QAction(this);
-  action->setText(tr("&Support Ticket..."));
-  action->setStatusTip(tr("Open a Support Ticket with Cyberbotics."));
+  action->setText(tr("&Support Ticket (Premier Service)..."));
+  action->setStatusTip(tr("Open a Support Ticket with Cyberbotics. This requires a subscription to the Premier Service"));
```
Comment (omichel) [inline src/webots/gui/WbMainWindow.cpp]:
```suggestion
  action->setStatusTip(tr("Open a Support Ticket with Cyberbotics. This requires a subscription to the Webots Premier Service."));
```

File: src/webots/gui/WbMainWindow.cpp [hunk 3]
```diff
@@ -908,23 +908,63 @@ QMenu *WbMainWindow::createHelpMenu() {
...
 
+  QMenu *followUsMenu = new QMenu(tr("&Keep informed"), this);
+
   action = new QAction(this);
-  action->setText(tr("Cyberbotics &Website..."));
-  action->setStatusTip(tr("Open the Cyberbotics website."));
-  connect(action, &QAction::triggered, this, &WbMainWindow::showCyberboticsWebsite);
-  menu->addAction(action);
+  action->setText(tr("Subscribe to the &Newsletter..."));
+  action->setStatusTip(tr("Keep informed about the last news about Webots following the Webots newsletter."));
```
Comment (omichel) [inline src/webots/gui/WbMainWindow.cpp]:
```suggestion
  action->setStatusTip(tr("Keep informed about the latest Webots news with the Webots newsletter."));
```

File: src/webots/gui/WbMainWindow.cpp [hunk 4]
```diff
@@ -908,23 +908,63 @@ QMenu *WbMainWindow::createHelpMenu() {
...
+  connect(action, &QAction::triggered, this, &WbMainWindow::openNewsletterSubscription);
+  followUsMenu->addAction(action);
+
+  action = new QAction(this);
+  action->setText(tr("Join the &Discord chanel..."));
```
Comment (omichel) [inline src/webots/gui/WbMainWindow.cpp]:
```suggestion
  action->setText(tr("Join the &Discord channel..."));
```
[chanel](https://www.chanel.com) is something different 😄

File: src/webots/gui/WbMainWindow.cpp [hunk 5]
```diff
@@ -908,23 +908,63 @@ QMenu *WbMainWindow::createHelpMenu() {
...
+  action->setStatusTip(tr("Join our live community on Discord."));
+  connect(action, &QAction::triggered, this, &WbMainWindow::openDiscord);
+  followUsMenu->addAction(action);
+
+  action = new QAction(this);
+  action->setText(tr("Follow Webots on &Twitter..."));
+  action->setStatusTip(tr("Keep informed about the last news about Webots on Twitter."));
```
Comment (omichel) [inline src/webots/gui/WbMainWindow.cpp]:
```suggestion
  action->setStatusTip(tr("Keep informed about the latest Webots news on Twitter."));
```

File: src/webots/gui/WbMainWindow.cpp [hunk 6]
```diff
@@ -908,23 +908,63 @@ QMenu *WbMainWindow::createHelpMenu() {
...
+  connect(action, &QAction::triggered, this, &WbMainWindow::openTwitter);
+  followUsMenu->addAction(action);
+
+  action = new QAction(this);
+  action->setText(tr("Subscribe to the Webots &YouTube chanel..."));
+  action->setStatusTip(tr("Take a look at the last Webots movies on YouTube."));
```
Comment (omichel) [inline src/webots/gui/WbMainWindow.cpp]:
```suggestion
  action->setStatusTip(tr("Watch the latest Webots movies on YouTube."));
```

**Comment by fabienrohrer** [comment]
@omichel I applied all your suggestions as-is.

👍

**Comment by fabienrohrer** [comment]
👍

**Pull request merged**